### PR TITLE
fix sticky header collapse, add scroll margin to content

### PR DESCRIPTION
**Pull request by gracegrimwood**
Fixes issue where anchor links would be covered by the navbar, and additionally fixes an issue where the navbar would disappear after scrolling the height of the screen (body size was locked to screen size, and navbar visibility was locked to the height of its parent element, which is the body).

### CODE REVIEW

**Review by SamBarker — APPROVED** [review]
LGTM

Thanks for all your css wrangling @gracegrimwood

**Comment by k-wall** [comment]
Thank you @gracegrimwood!

File: _sass/kroxylicious.scss
Comment (gracegrimwood) [reply _sass/kroxylicious.scss:41]: The navbar is actually closer to `3.5rem` but once I accounted for drop-shadow and line spacing `5rem` ended up being what looked best. I used `rem` and not `em` so that everything had consistent spacing from the top of the page, otherwise smaller headings would have been closer to the top and larger headings would have been further. I avoided fixed `px`, `vh`, and `%` units because it broke things in new and interesting ways on phone screens.

Comment (SamBarker) [reply _sass/kroxylicious.scss:41]: Ohh that was a lot more funky than I expected thank you!

**Review by k-wall — APPROVED** [review]

File: _sass/kroxylicious.scss
Comment (k-wall) [reply _sass/kroxylicious.scss:41]: Yeah, I'm so glad I didn't try that rabbit hole myself!

**Comment by gracegrimwood** [comment]
Cheers! I don't have write access to your branch @k-wall, I'll let you merge this one :)

**Pull request merged**
